src/Components/WebView/WebView/test/wwwroot/webviewtesthost.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <title>PhotinoTestApp</title>
+    <base href="/subdir/" />
+    <link href="css/app.css" rel="stylesheet" />
+</head>
+
+<body>
+    <root>Loading...</root>
+
+    <!-- Explicit display:none required so StartupErrorNotificationTest can observe it change -->
+    <div id="blazor-error-ui" style="display: none;">
+        An unhandled error has occurred.
+        <a href class='reload'>Reload</a>
+        <a class='dismiss' style="cursor: pointer;">🗙</a>
+    </div>
+
+    <script src="_framework/blazor.webview.js"></script>
+
+    <script>
```

Comment:
This script block is how the C# app sends commands to the WebView to do things (like, click a button), and also how C# queries the JS code to evaluate the state of the test (like, get me the value of an HTML element).